src/pages/components/WebView.qml

```diff
@@ -150,7 +159,7 @@ WebContainer {
                 }
 
                 if (loaded) {
-                    webView.privateMode ? grabThumbnail() : grabToFile()
+                    webView.privateMode ? grabThumbnail(getThumbnailSize()) : grabToFile(getThumbnailSize())
```

Comment:
Update also FavoriteGrid.qml approx line 192. That should use always grabThumbnail(). Size could be something like Screen.width / 3 for width and height for favorites.